[FIX] l10n_ar_ux: cambio en vista l10n_ar.custom_header

pablohmontenegro opened the pull request:
While accessing /my/invoices, and having installed l10n_ar, we have a warning that the t-esc is not understood. The inherited attribute then does not work. Let's replace it for a t-out. Ticket Adhoc side: 79024

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/904).

pablohmontenegro committed:
[FIX] l10n_ar_ux: cambio en vista l10n_ar.custom_header.
While accessing /my/invoices, and having installed l10n_ar, we have a warning that the t-esc is not understood. The inherited attribute then does not work. Let's replace it for a t-out. Ticket Adhoc side: 79024
Ticket: 79024

jjscarafia commented:
mezclamos igual porque esto debería arreglar runbot


@roboadhoc r+

roboadhoc commented:
@jjscarafia you may want to rebuild or fix this PR as it has failed CI.

jjscarafia commented:
@roboadhoc override=ci/runbot-oba
